logic/src/commonMain/kotlin/com/wire/kalium/logic/data/message/ProtoContentMapper.kt

```diff
@@ -83,9 +86,21 @@ class ProtoContentMapperImpl(
 
     override fun decodeFromProtobuf(encodedContent: PlainMessageBlob): ProtoContent {
         val genericMessage = GenericMessage.decodeFromByteArray(encodedContent.data)
-
-        kaliumLogger.d("Received message $genericMessage")
         val protobufModel = genericMessage.content
+        when (protobufModel) {
+            is GenericMessage.Content.Text, is GenericMessage.Content.Edited -> {
```

Comment:
yeah actually the initial plan was to remove the content all along, but I understood from @tmspzz that some content are okay to log, but actually you have a point there, even with a small info it can tell a lot about the conversation